src/ShiftedCompositeNormL1.jl

```diff
@@ -0,0 +1,85 @@
+export ShiftedCompositeNormL1
+
+mutable struct ShiftedCompositeNormL1{
```

Comment:
This needs some documentation to explain that $c$ will be linearized; otherwise, we would expect the shifted function to be $s \to \Vert c(x+s) \Vert_1$.

Reply:
See MaxenceGollier:affineNormL1@bb3faaa95b8e4d59bbb29ce5885779a10c5cfc0a